monai/csrc/utils/monai_utils.h

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I feel `monai_utils` may not be a suitable file name, because we are MONAI project..
Why not just using `utils.h` under `monai/csrc/`?

Thanks.

Reply:
sure I'll rename monai_utils.h -> common_utils.h. there'll be some other shared utils under the `csrc/utils/` folder